src/atoms/Link/Link.js

```diff
@@ -50,11 +50,15 @@ const Link = (props) => {
 
   if (isHostnameMatch(href)) {
     return (
-      <RouterLink href={pathname(href)} as={as}>
-        <a {...rest} className={fullClassName} data-testid="hostnameLink">
-          {children}
-        </a>
-      </RouterLink>
+      <Link
+        {...rest}
+        href={pathname(href)}
+        as={as}
```

Comment:
Not seeing the as prop as supported anymore  https://nextjs.org/docs/pages/api-reference/components/link.. Am I missing something?

Reply:
The **as prop** is deprecated and is no longer required in Next v9.5.3 and above.